File: Business/eUniversity.Business.ViewModels/Profile/GeneralInfoSectionViewModel.cs
namespace eUniversity.Business.ViewModels.Profile
{
    /// <summary>
    /// View model for gerental info section
    /// </summary>
    public class GeneralInfoSectionViewModel
    {
        public string UserName { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string BithDayDisplay { get; set; }
        public bool? IsMale { get; set; } //todo: change to enum
        public string Comment { get; set; }
    }
}

File: Web/eUniversity.Web/Scripts/viewModels/profile/generalInfoSectionViewModel.js
function GeneralInfoSectionViewModel(serverModel) {
    var self = this;
    self.UserName = ko.observable();
    self.FirstName = ko.observable();
    self.LastName = ko.observable();
    self.LastName = ko.observable();
    self.BithDayDisplay = ko.observable();
    self.IsMale = ko.observable();
    self.Comment = ko.observable();
    
    ko.mapping.fromJS(serverModel, {}, self);
}